src/scenes/home/Home.js

```diff
@@ -1,49 +1,60 @@
 import React, { Component } from "react";
-import { View } from "react-native";
+import { View, TouchableOpacity, Image } from "react-native";
 import ListTask from "./ListTasks";
 import styles from "./Home.styles";
 import colors from "src/helpers/Colors";
 import { Route } from "src/helpers/Route";
-
-
-let data = [
-  {
-    title: 'task1',
-    description: 'description of task1',
-    id: '1',
-    done: false,
-  },
-  {
-    title: 'task2',
-    description: 'description of task2',
-    id: '2',
-    done: false,
-  },
-  {
-    title: 'task3',
-    description: 'description of task3',
-    id: '3',
-    done: false,
-  },
-];
+import iconPlus from '../../../assets/iconPlus.png';
 
 export class Home extends Component {
-  static navigationOptions = {
-    title: "Home",
-    headerStyle: {
-      backgroundColor: colors.clearBlue
-    },
-    headerTintColor: colors.white,
-    headerTitleStyle: styles.text
+  static navigationOptions = ({ navigation }) => {
+    const buttonPlus = (
+      <TouchableOpacity style={styles.plus} onPress={() => {
```

Comment:
Its better also to put this function in a variable and avoid to put arrows in the jsx.
Anyways, you are using eslint right?